### PR TITLE
Add metadata and changelog for 0.2.6

### PR DESCRIPTION
**Pull request by HeroicKatora**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## v0.2.6 (2022-05-09)
+
+- Further fix to allow usage in WASM target.
```
Comment (fintelia) [inline CHANGELOG.md:7]: nit: "Another fix..." sounds slightly more natural to me but either works

**Review by fintelia — APPROVED** [review]

**Comment by gents83** [comment]
Does image Cargo.toml need to be updated to version 0.2.6 to retrieve WASM fix (it refers 0.2.1 now)?

**Comment by HeroicKatora** [comment]
`cargo update -p jpeg-decoder --precise 0.2.6` should suffice.

**Comment by gents83** [comment]
Even if in the end I'm just using directly the image crate and not the jpeg-decoder one?

**Comment by HeroicKatora** [comment]
Yes.

**Pull request merged**